types/node/index.d.ts

```diff
@@ -123,7 +123,7 @@ interface NodeModule {
     id: string;
     filename: string;
     loaded: boolean;
-    parent: NodeModule | null;
+    parent: NodeModule | undefined;
```

Comment:
I can remember that `parent` may be also `null` in some cases but I can't remember the exact use case.
Please adapt also the `class NodeJS.Module` accordingly.